plugins/catalog-react/src/components/UserListPicker/useOwnedEntitiesCount.ts

```diff
@@ -75,15 +78,19 @@ export function useOwnedEntitiesCount() {
 
   useDeepCompareEffect(() => {
     // context contains no filter, wait
-    if (Object.keys(filter).length === 0) {
+    if (Object.keys(catalogFilters).length === 0) {
```

Comment:
same as the other hook

```suggestion
    if (Object.keys(catalogFilters.filter).length === 0) {
```